tools/issue-labeler/Documentation/README.md

```diff
@@ -0,0 +1,74 @@
+
+
+## Intro to issue labeling
```

Comment:
We should kill the entire `Documentation` folder.   It's all about the .NET process and doesn't apply to us as it exists today.